scripts/src/internal/services.ts

```diff
@@ -154,14 +156,14 @@ export async function paymentComplete(payment: CompletedPayment, logger: LoggerI
 				return;
 			} else {
 				order.value = asset.asOrderValue();
-				asset.ownerId = order.sender!.id;
+				asset.ownerId = order.user.id;
```

Comment:
don't you have sender anymore?

Reply:
there's a `sender`, but when there's one user you can just user that